indexer/services/ender/src/caches/candle-cache.ts

```diff
@@ -13,6 +15,10 @@ import _ from 'lodash';
 let candlesMap: CandlesMap = {};
 
 export async function startCandleCache(txId?: number): Promise<void> {
+  const latestBlockTime: IsoString = await BlockTable.getLatest({ txId })
```

Comment:
then / catch is no longer preferred styling. Sth like

```
let latestBlockTime: IsoString;
try {
  const latestBlock = await BlockTable.getLatest({ txId });
  latestBlockTime = latestBlock.time;
} catch (error) {
  // Log the error since this is an unexpected state
  logger.Error(........ using current date as fall back)
  latestBlockTime = new Date().toISOString();
}
```

Also don't silently fall back to current timestamp